app/src/main/java/ch/epfl/qedit/view/edit/EditPreviewFragment.java

```diff
@@ -37,15 +36,7 @@ public View onCreateView(
 
         // Get the ViewModel and fix what happens when the focused question changes
         model = new ViewModelProvider(requireActivity()).get(EditionViewModel.class);
-        model.getFocusedQuestion()
-                .observe(
-                        getViewLifecycleOwner(),
-                        new Observer<Integer>() {
-                            @Override
-                            public void onChanged(Integer index) {
-                                onQuestionChanged(index);
-                            }
-                        });
+        model.getFocusedQuestion().observe(getViewLifecycleOwner(), this::onQuestionChanged);
```

Comment:
this is cool, i didn't knew this syntax

Reply:
Yeah it's nice! The operator is called 'method reference operator'.